### ISSUE TITLE
[req] TurkSeed

### ISSUE THREAD

**Issue by garfield69** [Jackett/Jackett#7753]
TurkSeed is a TURKISH Private Torrent Tracker for MOVIES / TV / GENERAL
https://turkseed.com/
unit3d 8.1.0
Registration Invitation only ATM
MR 0.5 MST 36h

**Comment by garfield69** [comment]
https://turkseed.com/ down for maint
![image](https://user-images.githubusercontent.com/6933240/93007245-ec92fe80-f5ba-11ea-8da0-eaff8aadac00.png)

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
https://trendfilm.net/ cloudflare Connection timed out Error code 522

**Comment by ilike2burnthing** [comment]
back up

**Comment by ilike2burnthing** [comment]
https://trendfilm.net/ redirects to https://forum.turkseed.com/ where credentials don't work.

https://turkseed.com/ shows the message:

```text
Dear members, as of December 31, 2023, we had to make changes to our trendfilm.net torrent site. You will download it from the direct link from our new site. You won't have to share. You will now be able to follow and download our new presentations from our new website. From now on, you will download our presentations by clicking the links. The memberships of our VIP members will continue where they left off. Once our VIP members become members and inform us of their membership names, the transactions will be carried out. Memberships will not be transferred. Please click to subscribe to our new site. https://forum.turkseed.com You will be directed to the new site in 10 seconds
```

I registered an account in case it was a translation issue, but no, they've moved to DDL. Removing indexer.

**Comment by garfield69** [comment]
v0.21.1465

**Comment by garfield69** [comment]
On Oct 2022 turkseed.com redirected to trendfilm.net
On Dec 2023 trendfilm became DDL only to the turkseed indexer was removed.
Now the turkseed.com domain is independent again, pointing to a new Unit3d 8.1.0 site.
Registration is currently Closed.
When it opens, or if an invite is provided, we can create a nre indexer for this domain.

**Comment by garfield69** [comment]
The login page states:
```
We do not accept new memberships, only old TURKSEED members can create their new passwords and log in to the site using Forgot My Password!!
```
I used my email address from my old account to try to get a password reset email, but nothing has arrived in either inbox or spam so far.
If anyone gets it working let me know and I will try again.

**Comment by garfield69** [comment]
Update: My password reset email has arrived and I'm in :-)
So I will begin to create the new indexer for this new site.

**Comment by garfield69** [comment]
New criteria: 
Accounts that do not download a 15GB torrent from our site within a month will receive a warning (will be closed)! 
Accounts that do not upload (seed, share) 20gb on our site within a month will receive a warning (will be closed)! 
So after I have created the indexer, I expect in a month time to have my account deleted, as I do not do DL/UL activity.

**Comment by AARON8888** [comment]
Yes, it´s a hard trackers with hard requirements. Also my account was purged, but not deleted, and I could recover it and create a new password to sign up again. I downloaded 21 gb torrent size with freeleech just in case I could pass the requirements which I consider absurb imo.

**Comment by ilike2burnthing** [comment]
The rules say twice that the MST is 36hrs, but then also once that it's 3 days 🤷 I will test and find out. Bumped the MR to cover the global requirement of 0.6

**Comment by garfield69** [comment]
v0.21.2503

**Comment by garfield69** [comment]
https://turkseed.com/ ERR_CONNECTION_TIMED_OUT

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://turkseed.com/ > Oops! An Error Occurred The server returned a "500 Internal Server Error".

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
v0.22.1760

**Comment by garfield69** [comment]
https://turkseed.com/ > Error 503: Service Unavailable!  Sorry, we are doing some maintenance. Please check back soon.
I believe they are upgrading to 9.0.8

**Comment by ilike2burnthing** [comment]
Back up, but now have the same unit3d id as Infinity, which is v8.3.6. Not sure if they've actually downgraded.

**Comment by ilike2burnthing** [comment]
https://github.com/Jackett/Jackett/issues/7753#issuecomment-2106062008

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/7753#issuecomment-2106062008

**Comment by garfield69** [comment]
online

**Comment by ilike2burnthing** [comment]
https://github.com/Jackett/Jackett/issues/7753#issuecomment-2106062008

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/7753#issuecomment-2106062008

**Comment by garfield69** [comment]
online
